src/components/FindWallet/WalletPersonasSidebar.tsx

```diff
@@ -0,0 +1,560 @@
+// Libraries
+import React, { useContext } from "react"
+import styled, { ThemeContext } from "styled-components"
+
+// Components
+import Icon from "../Icon"
+
+// Icons
+import BuyCrypto from "../../assets/wallets/buy_crypto.svg"
+import ENSSupport from "../../assets/wallets/ens_support.svg"
+import ERC20Support from "../../assets/wallets/erc_20_support.svg"
+import GasFeeCustomization from "../../assets/wallets/gas_fee_customization.svg"
+import HardwareSupport from "../../assets/wallets/hardware_support.svg"
+import Hardware from "../../assets/wallets/hardware.svg"
+import Layer2 from "../../assets/wallets/layer_2.svg"
+import NFTSupport from "../../assets/wallets/nft_support.svg"
+import NonCustodial from "../../assets/wallets/non_custodial.svg"
+import OpenSource from "../../assets/wallets/open_source.svg"
+import RPCImporting from "../../assets/wallets/rpc_importing.svg"
+import Staking from "../../assets/wallets/staking.svg"
+import WalletConnect from "../../assets/wallets/walletconnect.svg"
+import ConnectDapps from "../../assets/wallets/connect_dapps.svg"
+import WithdrawCrypto from "../../assets/wallets/withdraw_crypto.svg"
+import Multisig from "../../assets/wallets/multisig.svg"
+import SocialRecover from "../../assets/wallets/social_recover.svg"
+import Swap from "../../assets/wallets/swap.svg"
+import Eip1559 from "../../assets/wallets/eip1559.svg"
+
+// Utils
+import { trackCustomEvent } from "../../utils/matomo"
+
+// Styles
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+
+  @media (max-width: ${(props) => props.theme.breakpoints.s}) {
+    padding: 1rem 1rem;
+  }
+`
+
+const FeatureListItem = styled.li<{
+  selected: boolean
+}>`
+  display: flex;
+  gap: 0.2rem;
+  font-size: 0.85rem;
+  line-height: 0.95rem;
+  margin: 0.1rem;
+  align-items: center;
+  p {
+    margin-bottom: 0;
+    color: ${(props) =>
+      props.selected === true
+        ? props.theme.colors.primary
+        : props.theme.colors.text};
+  }
+
+  svg {
+    width: 28px;
+    height: 28px;
+    path {
+      fill: ${(props) => props.theme.colors.text};
+      stroke: ${(props) => props.theme.colors.text};
+    }
+  }
+`
+
+const Persona = styled.div<{
+  selected: boolean
+  isDark: boolean
+}>`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 1.5rem;
+  background: ${(props) =>
+    props.selected === true
+      ? props.isDark === true
+        ? props.theme.colors.primary900
+        : props.theme.colors.primary200
+      : props.isDark === true
+      ? props.theme.colors.black400
+      : props.theme.colors.primary100};
```

Comment:
`selected` and `isDark` are both typed as `boolean` so the `=== true` is not needed. 
```suggestion
  background: ${({ selection, isDark, theme }) =>
    selected
      ? isDark
        ? theme.colors.primary900
        : theme.colors.primary200
      : isDark
      ? theme.colors.black400
      : theme.colors.primary100};
```

Similar pattern below. Just syntactic sugar, so doesn't need to block, but just noting.